### PR TITLE
feat: its deploy

### PR DESCRIPTION
**Pull request by Foivos**

### CODE REVIEW

File: temp-arguments.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = [
```
Comment (milapsheth) [inline temp-arguments.js:1]: add this to gitignore

File: temp-arguments.js [hunk 2]
```diff
@@ -0,0 +1 @@
+module.exports = ['0x1a920B29eBD437074225cAeE44f78FC700B27a5d'];
```
Comment (milapsheth) [inline temp-arguments.js:1]: add to gitignore

File: info/testnet.json [hunk 3]
```diff
@@ -61,6 +61,26 @@
...
+          "standardizedTokenDeployer": "0xFF3de2ee3DF0cb25518bE948990902d4A17DE996",
+          "linkerRouterImplementation": "0x43c3d08f03aa43A8f49e98c9Ab9EB733ce65B11F",
+          "linkerRouter": "0xf4d6Bbc48425c7D8F2f9a0473bb12144fd9Cb11E",
+          "tokenManagerImplementations": [
+            "0x9237cB58050ED56d347899b168d7DdfB7541d188",
+            "0x1B1dE578b064FFF7A8528Ecf28c4dcA9Dd054767",
+            "0xaDDD1630d5F66d74C8dA6a5a4609bBb2cE90aBA4"
+          ],
+          "implementation": "0x5a6257d128CcBe796Dc37C875829b2FF39E5425E",
+          "address": "0xA4c905B10E15899FDD8558cf768E120372C6560E"
```
Comment (milapsheth) [inline info/testnet.json:83]: missing salt, deployer, operator args

**Review by milapsheth — APPROVED** [review]

**Review by deanamiel — APPROVED** [review]

**Review by milapsheth — APPROVED** [review]

File: evm/deploy-its.js [hunk 4]
```diff
@@ -0,0 +1,250 @@
+require('dotenv').config();
+
+const { getCreate3Address } = require('@axelar-network/axelar-gmp-sdk-solidity');
+const { deployContract, deployCreate3, isAddressArray, writeJSON } = require('./utils');
+const {
+    Wallet,
+    Contract,
+    getDefaultProvider,
+    utils: { defaultAbiCoder, isAddress, keccak256 },
+} = require('ethers');
```
Comment (milapsheth) [inline evm/deploy-its.js:10]: why not use hardhat here? seems to be deviating from other scripts

**Comment by milapsheth** [comment]
can you update the README with precise setup/commands to run to build?

File: evm/deploy-its.js (hunk 4 above)
Comment (deanamiel) [reply evm/deploy-its.js:10]: seems like a lot of the other deployment scripts are using ethers as well, can change this for all scripts in a separate PR

**Pull request merged**